tests/integration/conftest.py

```diff
@@ -378,13 +378,13 @@ def deactivated_scm_step_run(scm_step_run):
 
 @pytest.fixture
 def scm_release(scm_pipeline_run):
-    scm_release = models.SCMRelease(name='Version 0.13.1',
-                                    from_hash='577fe3f6a091aa4bad996623b1548b87f4f9c1f8',
-                                    to_hash='a49954f060b1b7605e972c9448a74d4067547443')
+    scm_release = models.SCMRelease.objects.filter(scm_pipeline_runs__pk__exact=scm_pipeline_run.pk).first()
 
     with username_on_model(models.SCMRelease, 'initial'):
+        scm_release.name = 'Version 0.13.1'
+        scm_release.from_hash = '577fe3f6a091aa4bad996623b1548b87f4f9c1f8'
+        scm_release.to_hash = 'a49954f060b1b7605e972c9448a74d4067547443'
         scm_release.save()
-        scm_release.scm_pipeline_runs.set([scm_pipeline_run])
```

Comment:
Had to update the fixtures because now releases are created automatically so we just update the release instead of creating it manually. (also below)